packages/website/src/globals.d.ts

```diff
@@ -3,6 +3,8 @@ import type * as ts from 'typescript';
 
 declare global {
   interface WindowRequire {
+    // We know it's an unsafe assertion. This is fine.
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-parameters
     <T extends unknown[]>(
       files: string[],
       success?: (...arg: T) => void,
```

Comment:
Because then it'd be annoying to type assert these:

https://github.com/typescript-eslint/typescript-eslint/blob/d07eb9e08039d421fa4da16931b0b8661851dc4e/packages/website/src/components/editor/loadSandbox.ts#L34-L37

I'll expand the comment a bit.